test/sanity/collections/issue1771EqualEqualHierarchy.wtest

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Same as above

Reply:
It seems to be many of these issues 1771 tests. I'll end the comment here to avoid the spam

Reply:
Renamed files